js/script.js

```diff
@@ -77,23 +77,19 @@ const parkingLotsStyle = {
 let locationTag = "";
 
 const urlAddress = window.location.href;
-
 if (urlAddress.indexOf("#parking-reform-map") > -1) {
-  const splitAdressArray = urlAddress.split("#");
-  if (splitAdressArray[1].indexOf("parking-reform-map") > -1) {
-    // if the tag is "parking-reform-map=....."
+  const [, splitAddressFragment] = urlAddress.split("#");
+  // If the tag is "parking-reform-map=....."
+  if (splitAddressFragment.indexOf("parking-reform-map") > -1) {

```

Comment:
This uses a JavaScript feature called "destructuring". It's simply another syntax for accessing elements of an array.

Here, I don't think this code is that much of an improvement. But the benefit of using the style guide overall is worth it